src/components/Button.jsx

```diff
@@ -0,0 +1,10 @@
+import React from "react";
+import styles from "./Button.module.css";
+
+const Button = ({size, variant, children}) => {
+  return (
+    <button className={`${styles.button} ${styles[variant]} ${styles[size]}`}>{children}</button>
```

Comment:
you can add onClick here and pass the function you want to execute when clicking on the button as a prop
such as `<button onClick={handleClick}>{children}</button>`